ziro-export-unity.ps1

```diff
@@ -43,10 +42,49 @@ function Execute-GetOnUnity {
         $Resources = $Response.$ResourceName
         $ResourcesArray += $Resources
     }
-
+    
     $JsonOutput = ConvertTo-Json $ResourcesArray
-    $JsonOutput | Out-File -FilePath $OutputFilePath
+
+    if ($OutputFileName) {
+        $OutputFilePath = "output-unity/" + $OutputFileName
+        $JsonOutput | Out-File -FilePath $OutputFilePath
+    }
+    
     return $JsonOutput | ConvertFrom-Json
+
+
+}
+
+function Export-Greetings {
+    param (
+        $Greetings,
+        [string]$CallHandlerId,
+        [string]$FolderName
+    ) 
+    $ProgressCount = 0
+    foreach ($Greeting in $Greetings) {
+        $ProgressCount++
+
+        $PlayWhat = [int]$Greeting.PlayWhat
+        $Enabled = [System.Convert]::ToBoolean($Greeting.Enabled)
+
+        if ($PlayWhat -eq 1 -and $Enabled -eq $true) {
+            $GreetingStreamFiles = Invoke-GetOnUnity $UnityHost ('/vmrest/handlers/callhandlers/' + $CallHandler.ObjectId + "/greetings/" + $Greeting.GreetingType + "/greetingstreamfiles") $Credential $null 'GreetingStreamFile'
+        
+            foreach ($GreetingStreamFile in $GreetingStreamFiles) {
+                $Url = $UnityHost + ('/vmrest/handlers/callhandlers/' + $CallHandler.ObjectId + "/greetings/" + $Greeting.GreetingType + "/greetingstreamfiles/" + $GreetingStreamFile.LanguageCode + "/audio")
+
+                $Headers = @{
+                    "Accept" = "application/json"
+                }
+
+                Invoke-RestMethod -Uri $Url -Headers $Headers -SkipCertificateCheck -Credential $Credential -OutFile ("output-unity/" + $FolderName + '/gr_' + $Greeting.GreetingType + "_" + $GreetingStreamFile.LanguageCode + ".wav")
```

Comment:
Question for my understanding: why is `-SkipCertificateCheck` used here?

Reply:
So in the original story https://github.com/Stack8/smacs/issues/16999 it says `The script should use HTTPS and ignore certificate issues (self-signed, outdated, wrong subject name...).` and that's what it is used for